rust/private/clippy.bzl

```diff
@@ -127,15 +145,16 @@ def _clippy_aspect_impl(target, ctx):
     ctx.actions.run(
         executable = ctx.executable._process_wrapper,
         inputs = compile_inputs,
-        outputs = [clippy_marker],
+        outputs = [clippy_out],
         env = env,
         tools = [toolchain.clippy_driver],
         arguments = args.all,
         mnemonic = "Clippy",
     )
 
     return [
-        OutputGroupInfo(clippy_checks = depset([clippy_marker])),
+        OutputGroupInfo(clippy_checks = depset([clippy_out])),
```

Comment:
I presume the infrastructure that reads ClippyInfo provider cannot read OutputGroupInfos, is that correct?

Reply:
It can read it, but since it's always present we can't depend on it in a useful way.